### ISSUE TITLE
Seven theme inactive primary tabs have insufficient color contrast

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#3185]
**Describe the defect**

Seven theme inactive primary tabs have insufficient color contrast. This causes our accessibility tests to fail. See: https://www.drupal.org/project/drupal/issues/2930508

**To Reproduce**
Run accessibility tests.

**Expected behavior**
Accessibility tests should pass.

**Screenshots**
<img width="1049" alt="Screen Shot 2020-10-15 at 2 33 17 PM" src="https://user-images.githubusercontent.com/101649/96183157-b87d6700-0ef3-11eb-9e87-5feaa2d8c270.png">
